### PR TITLE
CNV-36170: Multi enable button

### PR DESCRIPTION
**Pull request by upalatucci**
MultiNetworkPolicies can be disabled by configuration.

Show the enable page in this case. Disable the button if the user has no permissions.

The page is on `/networking.k8s.io~v1~NetworkPolicy/enable-multi` as all pages under `k8s.cni.cncf.io~v1beta1~MultiNetworkPolicy` are disabled by the console because the resource does not exist yet. 

Screenshot from 2024-03-27 12-27-40

In case the user does not have permissions:
Screenshot from 2024-03-27 12-27-21

Screenshot from 2024-03-27 12-27-18

### CODE REVIEW

File: src/utils/models/index.ts [hunk 1]
```diff
@@ -40,3 +40,19 @@ export const MultiNetworkPolicyModel: K8sModel = {
   namespaced: true,
   plural: 'multi-networkpolicies',
 };
+
+export const NetworkOperatorModel: K8sModel = {
+  abbr: 'N',
+  apiGroup: 'operator.openshift.io/v1',
```
Comment (avivtur) [inline src/utils/models/index.ts:46]: is the `/v1` at the end intentional?

File: src/utils/models/index.ts [hunk 2]
```diff
@@ -40,3 +40,19 @@ export const MultiNetworkPolicyModel: K8sModel = {
...
+  apiVersion: 'v1',
+  id: 'Network',
+  kind: 'Network',
+  label: 'network',
+  // t('network')
+  labelKey: 'network',
+  labelPlural: 'Networks',
+  // t('networks')
+  labelPluralKey: 'networks',
```
Comment (avivtur) [inline src/utils/models/index.ts:55]: is it possible to use directly to strings?

File: src/views/networkpolicies/list/hooks/useIsMultiEnabled.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { modelToGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { NetworkConfigModel } from '@views/nads/form/utils/constants';
+
+const useIsMultiEnabled = () => {
```
Comment (avivtur) [inline src/views/networkpolicies/list/hooks/useIsMultiEnabled.tsx:5]: Please add type here

File: src/views/networkpolicies/list/EnableMultiPage.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  EmptyStateFooter,
+  EmptyStateHeader,
+  Tooltip,
+} from '@patternfly/react-core';
+import { useLastNamespacePath } from '@utils/hooks/useLastNamespacePath';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import { MultiNetworkPolicyModel } from '@utils/models';
+import { NetworkConfigModel } from '@views/nads/form/utils/constants';
+
+import '@styles/list-management-group.scss';
```
Comment (avivtur) [inline src/views/networkpolicies/list/EnableMultiPage.tsx:25]: what is this for?

File: src/views/networkpolicies/list/EnableMultiPage.tsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+          op: 'replace',
+          path: '/spec/useMultiNetworkPolicy',
+          value: true,
+        },
+      ],
+      model: NetworkConfigModel,
+      resource: networkClusterConfig,
+    }).then(() => {
+      navigate(`/k8s/${lastNamespacePath}/${modelToRef(MultiNetworkPolicyModel)}`);
+      location.reload();
```
Comment (avivtur) [inline src/views/networkpolicies/list/EnableMultiPage.tsx:61]: ??

**Review by avivtur — COMMENTED** [review]
Nice :) 
Added some comments

File: src/views/networkpolicies/list/EnableMultiPage.tsx (hunk 5 above)
Comment (upalatucci) [reply src/views/networkpolicies/list/EnableMultiPage.tsx:61]: If we go to that page just after the config change, we will see 404 from the console. model does not exist yet. Reloading we will see the page

File: src/views/networkpolicies/list/EnableMultiPage.tsx (hunk 4 above)
Comment (upalatucci) [reply src/views/networkpolicies/list/EnableMultiPage.tsx:25]: it's on all list pages. It add some style that is on utils/style

File: src/utils/models/index.ts (hunk 1 above)
Comment (upalatucci) [reply src/utils/models/index.ts:46]: no thanks i'll fix that

**Review by avivtur — APPROVED** [review]

**Pull request merged**